dist/source/repo-creator/choose-name.ts

```diff
@@ -2,14 +2,21 @@ import { autoinject } from 'aurelia-dependency-injection';
 import { computedFrom } from 'aurelia-binding';
 import { Router } from 'aurelia-router';
 import { EventAggregator } from 'aurelia-event-aggregator';
+import { Validation } from 'aurelia-validation';
 
 @autoinject()
 export class ChooseName {
 	constructor(
 		private router: Router,
-		private eventAggregator: EventAggregator
-	) {}
-	
+		private eventAggregator: EventAggregator,
+		protected validation: Validation
+	) {
+
+		this.validation = validation.on(this)
```

Comment:
The assignment here isn't necessary.  `this.validation` already equals `validation` before the constructor is called.  Since this is a fluent API, call calls to `on`, `ensure` and `isNotEmpty` all return the same object so this assignment is the same as:
`this.validation = this.validation`
or
`this.validation = validation`